contribute/en/Style-and-Tone.md

```diff
@@ -83,17 +83,19 @@ Refer to UI labels in inverted commas (eg 'click on the “mute” button')
 
 Jamulus is “Free and Open Source (FOSS)” (not “free software” or “open source”)
 
-“Channel” The audio signal as part of a mix. “Mute a channel”, “Maximum number of channels”, “Group channels together” (not “Mute a person” because one person might be using multiple channels).
+“Channel” The audio signal as part of a mix. “Mute a channel”, “Maximum number of channels”, “Group channels together”.
```

Comment:
I call these _connections_. Each connection has one or two _channels_ (mono or stereo).

Reply:
My view is based on the code.  For me, in Jamulus terms "Channel" refers to what CChannel does - wraps a socket and exchanges data with another Jamulus instance.  That includes both audio and control data.  The style guide, however, isn't for code developers.  It's for people documenting Jamulus, so that such documentation is consistent.

Where some context - such as MIDI - makes the normal usage of Channel (with a capital letter - I think we should specify that) unclear, then the author needs to make it plain, as appropriate.

I'm not sure the JSON-RPC documentation - as the feature has been considered experimental - has been subject to the Style Guide.

Reply:
> I'm not sure the JSON-RPC documentation - as the feature has been considered experimental - has been subject to the Style Guide.

Currently it is, but I'd like to get it more mainstream in Jamulus 4. So we should be consistent there too.